Aspose.Slides.WebExtensions.Tests/TestDemo.cs

```diff
@@ -0,0 +1,61 @@
+using Aspose.Slides;
+using Aspose.Slides.Export.Web;
+using Aspose.Slides.WebExtensions;
+using NUnit.Framework;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Xml;
+
+namespace Aspose.Slides.WebExtensions.Tests
+{
+    public class TestDemo
+    {
+        private string TemplatePath = null;
+        private string PresentationFilePath = null;
+        private string OutputPath = null;
+        private string RootDirectory = null;
+
+        [SetUp]
+        public void Setup()
+        {
+            RootDirectory = TestContext.CurrentContext.TestDirectory;
+            PresentationFilePath = Path.Combine(RootDirectory, "TestData", "demo.pptx");
+            TemplatePath = Path.Combine(RootDirectory, "templates", "single-page");
+            OutputPath = Path.Combine(RootDirectory, "single-page-demo-output");
+
+            using (Presentation pres = new Presentation(PresentationFilePath))
+            {
+                WebDocument document = pres.ToSinglePageWebDocument(TemplatePath, OutputPath);
+                document.Save();
+            }
+        }
+        static string FixVideoTag(string input)

```

Comment:
@polonair ,

Why this method is needed? (XPath related? not sure that understand)

Reply:
Our code generates this:
```html
        <video style="left: 436px; top: 1796px; width: 481px; height: 270px;" controls>
            <source src="media/video0.mp4" type="video/mp4">
            Your browser does not support the video tag.
        </video>
```
This is valid html, but not valid xml because attribute `controls` has no value, and tag `<source>` has no close tag. So, method `FixVideoTag` fixes it and make valid xml code